invesalius/gui/task_navigator.py

```diff
@@ -1428,6 +1498,27 @@ def OnMenuSetColor(self, evt):
 
             Publisher.sendMessage('Set new color', index=index, color=color_new)
 
+    def OnMenuSendTarget2Robot(self, evt):
+        if isinstance(evt, int):
+           self.lc.Focus(evt)
+
+        robot = self.markers[self.lc.GetFocusedItem()].robot
+        head = self.markers[self.lc.GetFocusedItem()].head
+
+        trans = tr.translation_matrix(robot[:3])
```

Comment:
Change the matrix calculation to another file?

Reply:
Yes, similar to what Olli-Pekka did, move them to where the other matrix computations are. Maybe the `bases` module?